corehq/apps/builds/templates/builds/edit_menu.html

```diff
@@ -161,6 +179,9 @@ <h3 class="panel-title">Menu Options</h3>
                             <td>
                                 <input type="checkbox" value="superuser-only" data-bind="checked: superuser_only" />
                             </td>
+                            <td class='j2me_enabled'>
+                                <input type="checkbox" value="j2me_enabled" data-bind="checked: j2me_enabled" />
```

Comment:
Oh my bad! i took checkmark as checkbox coz I thought in that way the intent of having a checkbox was to make it configurable where the version check was like a suggestion from builds and its files. I over thought it i guess. Would change it as you suggested.
Thanks 👍